### PR TITLE
Add stream and blob availability commands

### PR DESCRIPTION
**Pull request by jackrobison**

### CODE REVIEW

**Comment by jackrobison** [comment]
Note that this doesn't redirect calls to the deprecated `get_availability` to the new `stream_availability`, I'll add that

File: lbrynet/core/Session.py [hunk 1]
```diff
@@ -136,6 +137,7 @@ def __init__(self, blob_data_payment_rate, db_dir=None,
         self.payment_rate_manager = None
         self.payment_rate_manager_class = payment_rate_manager_class or NegotiatedPaymentRateManager
         self.is_generous = is_generous
+        self.single_peer_downloader = SinglePeerDownloader()
```
Comment (lyoshenka) [inline lbrynet/core/Session.py]: why does a session always have a single-peer downloader? this functionality is unnecessary in 99% of cases. it should only be on when we're gonna use it

File: lbrynet/core/Session.py [hunk 2]
```diff
@@ -318,6 +320,7 @@ def _setup_other_components(self):
 
         dl = defer.DeferredList([d1, d2], fireOnOneErrback=True, consumeErrors=True)
         dl.addCallback(lambda _: self.blob_tracker.start())
+        dl.addCallback(lambda _: self.single_peer_downloader.setup(self.wallet, self.blob_manager))
```
Comment (lyoshenka) [inline lbrynet/core/Session.py]: same comment as above

File: lbrynet/core/Wallet.py [hunk 3]
```diff
@@ -27,8 +27,10 @@
 from lbrynet.core.sqlite_helpers import rerun_if_locked
 from lbrynet.interfaces import IRequestCreator, IQueryHandlerFactory, IQueryHandler, IWallet
 from lbrynet.core.client.ClientRequest import ClientRequest
-from lbrynet.core.Error import RequestCanceledError, InsufficientFundsError, UnknownNameError
+from lbrynet.core.Error import InsufficientFundsError, UnknownNameError
 from lbrynet.core.Error import UnknownClaimID, UnknownURI, NegativeFundsError, UnknownOutpoint
+from lbrynet.core.Error import DownloadCanceledError, RequestCanceledError
```
Comment (lyoshenka) [inline lbrynet/core/Wallet.py]: shouldn't  all these `from lbrynet.core.Error import ...` lines be one line? or is this about having lines that are too long?

File: lbrynet/daemon/Daemon.py [hunk 4]
```diff
@@ -2920,79 +2920,146 @@ def jsonrpc_routing_table_get(self):
         return self._render_response(result)
 
     @defer.inlineCallbacks
-    def jsonrpc_get_availability(self, uri, sd_timeout=None, peer_timeout=None):
+    def jsonrpc_blob_availability(self, blob_hash, search_timeout=None, blob_timeout=None):
```
Comment (lyoshenka) [inline lbrynet/daemon/Daemon.py]: so `get_availability` isn't just deprecated here, its removed, right?

File: lbrynet/core/Wallet.py (hunk 3 above)
Comment (jackrobison) [reply lbrynet/core/Wallet.py]: >is this about having lines that are too long?

right

**Pull request merged**